lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -136,7 +136,7 @@ import qualified Data.Set as Set
 data Wallet s t where
     Wallet :: (IsOurs s, NFData s, Show s, TxId t)
         => UTxO -- Unspent tx outputs belonging to this wallet
-        -> Set Tx -- Pending transactions
+        -> Set Tx -- Pending outgoing transactions
```

Comment:
Note: there's no such thing as pending incoming transactions ...

Reply:
I know that, but already had to explain it once, so updated the comment.

Reply:
I think the extra clarity is worth it. :+1:

Reply:
Sure, I was just pointing that out :)